--- atinject-1/build.sh~	2009-10-14 01:36:31.000000000 +0200
+++ atinject-1/build.sh	2014-06-09 08:21:06.686274678 +0200
@@ -10,8 +10,8 @@
 mkdir -p build/tck/dist
 
 # Compile classes.
-javac -g -d build/classes `find src -name *.java`
-javac -g -classpath build/classes:lib/junit.jar -d build/tck/classes \
+javac -source 1.5 -target 1.5 -g -d build/classes `find src -name *.java`
+javac -source 1.5 -target 1.5 -g -classpath build/classes:lib/junit.jar -d build/tck/classes \
 	`find tck -name *.java`
 
 FOOTER="<font size='-1'>Copyright (C) 2009 <a href='http://code.google.com/p/atinject/'>\
